src/common/options.cc

```diff
@@ -2617,6 +2617,10 @@ std::vector<Option> get_global_options() {
     .set_default(0)
     .set_description(""),
 
+    Option("osd_pg_epoch_max_lag_factor", Option::TYPE_FLOAT, Option::LEVEL_ADVANCED)
```

Comment:
Shouldn't this get placed with the osd_map_cache_size, since it's a modifier on that?
And it should use the .add_see_also() stuff?

Reply:
fixed